subsys/bootloader/cmake/debug_keys.cmake

```
@@ -31,8 +31,16 @@ if (DEFINED SB_SIGNING_KEY_FILE)
   set(SIGNATURE_PRIVATE_KEY_FILE ${SB_SIGNING_KEY_FILE})
 endif()
 
-# Lastly, check if debug keys should be used.
+# Check if debug sign key should be generated.
 if( "${CONFIG_SB_SIGNING_KEY_FILE}" STREQUAL "")
+  message(WARNING "
+    --------------------------------------------------------
+    --- WARNING: Using generated NSIB key, it should not ---
```

Should we explicitly state that it is used for signing? To make it differ from the one for the public keys.

---

Well the private part is used for signing and the public of verifying, so calling it a keypair might be the best
```suggestion
    --- WARNING: Using generated NSIB public/private key-pair, it should not ---
```